products/products.css

```diff
@@ -0,0 +1,158 @@
+@import url("https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.1/font/bootstrap-icons.css");
+
+:root{
+    --btn-color:#2563eb;
+    --red-color:#d946ef;
+}
+body{
+    margin:0;
+    padding: 0;
+    height: 100vh;
+    box-sizing: border-box;
+    background-color: var(--bg-color);
+}
+
+.app-container{
+    display: grid;
+    grid-template-columns: repeat(5, 1fr);
```

Comment:
I would suggest keeping keeping a max of 4 cards in one row, and increasing the card's width. 
Also would look good if there was a hover state on "Add to Cart" button.